samples/subsys/nsettings/README.rst

```diff
@@ -0,0 +1,68 @@
+.. _nvs-sample:
+
+NVS: Non-Volatile Storage
+#########################
+
+Overview
+********
+
+This is a simple application demonstrating use of the NVS
+module for non-volatile (flash) storage.  In this application,
+a counter is incremented on every reboot and stored in flash,
+the application reboots, and the reboot counter data is retrieved.
+
+Requirements
+************
+
+* A board with flash support
+
+Building and Running
+********************
+
+This sample can be found under :file:`samples/subsys/nvs` in the Zephyr tree.
```

Comment:
This should be referring to the sample here in ``nsettings``, not the sample in ``nvs``